app/views/content/landing/still-time-to-apply/_content.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-720 centered">
+      <p class="inset">
+        Every day, people are being accepted onto teacher training courses starting this September. Places fill up quickly at this time of year so don’t miss your opportunity to join a course while there’s still time to apply.
+      </p>
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Search for a teacher training course</h2>
+
+      <p>Teacher training courses usually take 9 months full-time, or 18 to 24 months part time.
```

Comment:
Also, not sure if this is just my review app playing up but this text seems a bit out of line with the rest? The other sections appear more centre aligned to me whereas this is more over to the left? (Could just be me though!)